lib/elasticity/aws_request.rb

```diff
@@ -138,30 +98,10 @@ def get_secret_key(secret)
       raise MissingKeyError, 'Please provide a secret key or set AWS_SECRET_ACCESS_KEY.'
     end
 
-    # (Used from RightScale's right_aws gem.)
-    # EC2, SQS, SDB and EMR requests must be signed by this guy.
-    # See: http://docs.amazonwebservices.com/AmazonSimpleDB/2007-11-07/DeveloperGuide/index.html?REST_RESTAuth.html
```

Comment:
This is the old Sigv2 signing code... note the date in the comment here: `2007-11-07`